src/collection.js

```diff
@@ -20,15 +25,47 @@ CKEDITOR.define( [ 'emittermixin', 'ckeditorerror', 'utils' ], function( Emitter
 		 * Creates a new Collection instance.
 		 *
 		 * @constructor
+		 * @param {Iterale} [items] Items to be added to the collection.
+		 * @param {String} [idName='id'] The name of the property which is considered to identify an item.
 		 */
-		constructor() {
+		constructor( items, idName ) {
```

Comment:
Actually... I'd choose a 3rd way – `Collection( options )` and chainable `add( item | items )`. That's best for single resp principle.